app/qml/onboarding/MMHowYouFoundUs.qml

```diff
@@ -41,6 +41,7 @@ Page {
     id: header
 
     title: listView.contentY > -30 * __dp ? root.headerTitle : ""
+    backVisible: false
```

Comment:
Should I completely remove the root page's `backClicked` signal too?

Reply:
lets keep it for now